frontend/tests/components/addMemberModal.ts

```diff
@@ -3,6 +3,7 @@ import { BaseComponent } from './baseComponent';
 
 const EMAIL_LABEL = 'Email';
 const ROLE_LABEL = 'Role';
+const CHECKBOX_LABEL = 'Invite';
```

Comment:
I'd like this to be `INVITE_LABEL`, as `CHECKBOX_LABEL` doesn't communicate what it is. (We might well add another checkbox later on).